src/main/java/com/google/cloud/resourcemanager/OperationInfo.java

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.services.cloudresourcemanager.model.Operation;
+import com.google.common.base.MoreObjects;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/** Google Resource Manager Operation metadata. */
+public class OperationInfo {
+
+  private final Boolean done;
+  private final Status error;
+  private final Map<String, Object> metadata;
+  private final String name;
+  private final Map<String, Object> response;
+
+  /**
+   * The Status type defines a logical error model that is suitable for different programming
+   * environments, including REST APIs and RPC APIs. It is used by gRPC. Each Status message
+   * contains three pieces of data: error code, error message, and error details.
+   */
+  static class Status {
+
+    private Integer code;
+    private List<Map<String, Object>> details;
+    private String message;
+
+    Status(Integer code, String message, List<Map<String, Object>> details) {
+      this.code = code;
+      this.message = message;
+      this.details = details;
+    }
+
+    /** Returns the status code. */
+    Integer getCode() {
+      return code;
+    }
+
+    /** Returns the error message. */
+    String getMessage() {
+      return message;
+    }
+
+    /** Return the list of messages that carry the error details. */
+    List<Map<String, Object>> getDetails() {
```

Comment:
Nested data structure types such as List<Map> usually suggest there's a more specific class that should be created instead. Doubly so when you're using Object as a type parameter. Here possibly a Detail or Details class.

Reply:
same here please check these field as well see https://github.com/googleapis/google-api-java-client-services/blob/master/clients/google-api-services-cloudresourcemanager/v1/1.30.1/com/google/api/services/cloudresourcemanager/model/Status.java#L52